packages/eslint-plugin/tests/rules/no-invalid-void-type.test.ts

```diff
@@ -162,7 +154,7 @@ ruleTester.run('allowInGenericTypeArguments: true', rule, {
         {
           messageId: 'invalidVoidNotReturnOrGeneric',
           line: 1,
-          column: 41,
+          column: 34,
```

Comment:
We still have two errors now, but they're in the same place. That seems wrong.

https://deploy-preview-5671--typescript-eslint.netlify.app/play#ts=4.8.3&sourceType=module&code=MYewdgzgLgBAhgJwSA7gcQKZgwglsAJhgF4YAeAFRgwA8osATCGANxFwZNfYYD4AKRAHMAXDAoBKErxgBvAFAwYoSCAA2GAHRqQQwQiESA3PIC+RoA&eslintrc=N4KABGBEBOCuA2BTAzpAXGUEKQAIBcBPABxQGNoBLY-AWhXkoDt8B6Jge1uYDcBDRgBNaPDpWFFS6KImjQO0SODABfECqA&tsconfig=N4KABGBEDGD2C2AHAlgGwKYCcDyiAuysAdgM6QBcYoEEkJemy0eAcgK6qoDCAFutAGsylBm3TgwAXxCSgA

Reply:
@JoshuaKGoldberg 
I fixed the bug and test!
Please check the below! 🙏

https://github.com/typescript-eslint/typescript-eslint/pull/5671/commits/5225063e5a5a13a6a831c49b16a43bf703bdee4a

https://github.com/typescript-eslint/typescript-eslint/pull/5671/commits/2ed96e1f5f84d749b1dfcf3b11a85abd9f0dded1